qa-include/app/users.php

```diff
@@ -665,6 +662,9 @@ function qa_get_user_avatar_html($flags, $email, $handle, $blobId, $width, $heig
 				break;
 			case 'local-default':
 				$html = qa_get_avatar_blob_html(qa_opt('avatar_default_blobid'), qa_opt('avatar_default_width'), qa_opt('avatar_default_height'), $size, $padding);
+				if (strlen($handle) == 0) {
```

Comment:
You can use `===`

Reply:
Why would I check for type when `strlen` always return an `int`?

Reply:
If `strlen` always return `int`, you can use it without stress ;) And seriously, it's better to always use this operator wherever we can

Reply:
My rule of thumb is: for operands that CAN be of more than one type (mixed) or null always use `===`. Otherwise, `==`, to avoid the unnecessary type-check